webapp/channels/src/components/admin_console/schema_admin_settings.tsx

```diff
@@ -37,23 +42,53 @@ import Setting from './setting';
 
 import './schema_admin_settings.scss';
 
-export default class SchemaAdminSettings extends React.PureComponent {
-    static propTypes = {
-        config: PropTypes.object,
-        environmentConfig: PropTypes.object,
-        setNavigationBlocked: PropTypes.func,
-        schema: PropTypes.object,
-        roles: PropTypes.object,
-        license: PropTypes.object,
-        editRole: PropTypes.func,
-        updateConfig: PropTypes.func.isRequired,
-        isDisabled: PropTypes.bool,
-        consoleAccess: PropTypes.object,
-        cloud: PropTypes.object,
-        isCurrentUserSystemAdmin: PropTypes.bool,
-    };
+type ConsoleAccess = {
+    read: Record<string, boolean>;
+    write: Record<string, boolean>;
+}
 
-    constructor(props) {
+type Props = {
+    schema: any;
```

Comment:
This one might be a bit complicated, but I think there is a type floating around for this somewhere.